samples/tab-deeplink/nodejs/Bots/DeepLinkTabsnode.js

```diff
@@ -1,31 +1,33 @@
-          var MessagingDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("topic2",2,"Messaging Extension",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,EntityID);
-          var AdaptiveCardDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("topic3",3,"Adaptive Card",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,EntityID);  
-        }       
-         else  if(context.activity.conversation.conversationType==="personal")
-         {
-          var BotsDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic1",1,"Bots",process.env.MicrosoftAppId);
-          var MessagingDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic2",2,"Messaging Extension",process.env.MicrosoftAppId);
-          var AdaptiveCardDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic3",3,"Adaptive Card",process.env.MicrosoftAppId);            
-         }
-         
-        await context.sendActivity({ attachments: [this.createAdaptiveCard(conversationType, BotsDeepLink,MessagingDeepLink,AdaptiveCardDeepLink)] }); 
+          const replyText = `Echo: ${ context.activity.text }`;  
+          let conversationType = context.activity.conversation.conversationType;
+          let ExtendedDeepLink = "";
+          
+          if(context.activity.conversation.conversationType==="channel")
+          {
+            var BotsDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("topic1",1,"Bots",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,process.env.Channel_Entity_Id);
+            var MessagingDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("topic2",2,"Messaging Extension",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,process.env.Channel_Entity_Id);
+            var AdaptiveCardDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("topic3",3,"Adaptive Card",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,process.env.Channel_Entity_Id);
+            ExtendedDeepLink = DeepLinkTabHelper.GetDeepLinkTabChannel("",4,"Extended Deeplink features",context.activity.channelData.teamsChannelId,process.env.MicrosoftAppId,process.env.Channel_Entity_Id);
+          }       
+          else  if(context.activity.conversation.conversationType==="personal")
+          {
+            var BotsDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic1",1,"Bots",process.env.MicrosoftAppId);
+            var MessagingDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic2",2,"Messaging Extension",process.env.MicrosoftAppId);
+            var AdaptiveCardDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("topic3",3,"Adaptive Card",process.env.MicrosoftAppId);            
+            ExtendedDeepLink = DeepLinkTabHelper.GetDeepLinkTabStatic("",4,"Extended Deeplink features",process.env.MicrosoftAppId);            
+          }
```

Comment:
Check for proper spacing between === and at other places as well like after if (...)